src/prpy/rave.py

```diff
@@ -235,3 +235,229 @@ def disable_padding(body, enable=False):
     for link in body.GetLinks():
         if link.GetName().startswith('padding_'):
             link.Enable(enable)
+
+
+def add_box(env, name, x_len=0.1, y_len=0.1, z_len=0.1):
+    """Add simple box kinbody to OpenRAVE environment"""
+    with env:
+       box = openravepy.RaveCreateKinBody(env, '')
+       success = box.InitFromBoxes(numpy.array([[0,0,0,x_len,y_len,z_len]]))
```

Comment:
I'm pretty sure `InitFromBoxes` takes half-extents, not extents. You should either rename the function arguments accordingly or add the factor of `2.0` in here.

Also, please add `@param` and `@return` annotations to the `__docstring__`.